src/basic/macro.h

```diff
@@ -5,7 +5,6 @@
 #include <errno.h>
```

Comment:
Hm? The commit title is outdated? An empty file for sys/param.h is not included in the first commit.